Stripe/StripeError+Internal.h

```diff
@@ -0,0 +1,17 @@
+//
+//  NSError+StripeInternal.h
+//  Stripe
+//
+//  Created by Ben Guo on 5/22/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface NSError (StripeInternal)
```

Comment:
Ah, to avoid the category linking dance? Maybe! It'd be a breaking change to `StripeError.h` though, so I'd prefer punting.